test/commands/metric/create_test.rb

```diff
@@ -25,8 +25,8 @@ class Metric::CreateTest < ActiveSupport::TestCase
     solution = create :concept_solution
     track = solution.track
     user = solution.user
-    remote_ip = '127.0.0.1'
-    country_code = 'US'
+    remote_ip = SecureRandom.compact_uuid
```

Comment:
I'm now using random IP to stub as to now overwrite the default stub that is setup in the test helper